mobile/examples/question_answering/android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="app_name">ORT QuestionAnswering</string>
+    <string name="perform_qa">GO</string>
```

Comment:
maybe a [->] button would make more sense then [GO] for perform QA?

as a reference: TFLite Q&A sample looks like this: https://github.com/tensorflow/examples/tree/master/lite/examples/bert_qa/android#tensorflow-lite-bert-question--answer-demo-application

Reply:
Done.